build-browser.js

```diff
@@ -0,0 +1,23 @@
+const pkg = require('./package.json');
+const rollup = require('rollup').rollup;
+const babel = require('rollup-plugin-babel');
+
+rollup({
+  entry: 'src/index.js',
+  plugins: [
+    babel({
+      presets: ['es2015-rollup'],
+      plugins: ['transform-node-env-inline'],
+      babelrc: false,
+    }),
+  ],
+}).then(bundle => (
+  bundle.write({
+    dest: 'validator.js',
+    format: 'umd',
+    moduleName: pkg.name,
```

Comment:
I believe this is what is causing issues when I tried to switch to es6 modules in an ember app - https://github.com/TryGhost/Ghost-Admin/pull/34

Please let me know if there is any way I can help resolve the issue.